### DISCUSSION TITLE
@formily/antd 的ArrayTable案例，点击`添加条目`按钮会白屏

### DISCUSSION THREAD

**Discussion by shinyruo177** [alibaba/formily#2280] [RFC]
https://antd.formilyjs.org/zh-CN/components/array-table

###步骤
Markup Schema 案例等，点击`添加条目`按钮 
![image](https://user-images.githubusercontent.com/40891772/136755162-68d9a20c-ed2a-4f78-be87-c5d73dfed0c0.png)

###现象 
页面空白，控制台报错
![image](https://user-images.githubusercontent.com/40891772/136755722-7a17f966-1b71-4c6c-8169-1f0e26141c0e.png)

**Comment by janryWang** [comment]
提个bug
